SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -84,37 +84,38 @@ bool FStartDeployment::Update()
 		const FString RuntimeVersion = SpatialGDKEditorSettings->GetSelectedRuntimeVariantVersion().GetVersionForLocal();
 		const uint16 RuntimeGRPCPort = SpatialGDKEditorSettings->GetDefaultReceptionistPort();
 
-		AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [LocalDeploymentManager, LaunchConfig, LaunchFlags, SnapshotName,
-																 RuntimeVersion, RuntimeGRPCPort] {
-			if (!GenerateWorkerJson())
-			{
-				return;
-			}
-
-			if (!GenerateWorkerAssemblies())
-			{
-				return;
-			}
-
-			FSpatialLaunchConfigDescription LaunchConfigDescription;
-
-			FWorkerTypeLaunchSection AutomationWorkerConfig;
-			AutomationWorkerConfig.WorkerTypeName = TEXT("AutomationWorker");
-			LaunchConfigDescription.AdditionalWorkerConfigs.Add(AutomationWorkerConfig);
-
-			if (!GenerateLaunchConfig(LaunchConfig, &LaunchConfigDescription, /*bGenerateCloudConfig*/ false))
-			{
-				return;
-			}
-
-			if (LocalDeploymentManager->IsLocalDeploymentRunning() || LocalDeploymentManager->IsDeploymentStarting()
-				|| LocalDeploymentManager->IsDeploymentStopping())
-			{
-				return;
-			}
-
-			LocalDeploymentManager->TryStartLocalDeployment(LaunchConfig, RuntimeVersion, LaunchFlags, SnapshotName, TEXT(""), RuntimeGRPCPort, nullptr);
-		});
+		AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask,
+				  [LocalDeploymentManager, LaunchConfig, LaunchFlags, SnapshotName, RuntimeVersion, RuntimeGRPCPort] {
+					  if (!GenerateWorkerJson())
+					  {
+						  return;
+					  }
+
+					  if (!GenerateWorkerAssemblies())
+					  {
+						  return;
+					  }
+
+					  FSpatialLaunchConfigDescription LaunchConfigDescription;
+
+					  FWorkerTypeLaunchSection AutomationWorkerConfig;
+					  AutomationWorkerConfig.WorkerTypeName = TEXT("AutomationWorker");
+					  LaunchConfigDescription.AdditionalWorkerConfigs.Add(AutomationWorkerConfig);
+
+					  if (!GenerateLaunchConfig(LaunchConfig, &LaunchConfigDescription, /*bGenerateCloudConfig*/ false))
+					  {
+						  return;
+					  }
+
+					  if (LocalDeploymentManager->IsLocalDeploymentRunning() || LocalDeploymentManager->IsDeploymentStarting()
+						  || LocalDeploymentManager->IsDeploymentStopping())
+					  {
+						  return;
+					  }
+
+					  LocalDeploymentManager->TryStartLocalDeployment(LaunchConfig, RuntimeVersion, LaunchFlags, SnapshotName, TEXT(""),
+																	  RuntimeGRPCPort, false, nullptr);
+				  });
```

Comment:
got some extra spaces sprinkled throughout here.

Reply:
This should all just be grep formatting which wasn't run when this file was committed last. Additionally there's one more parameter that's being sent to TryStartLocalDeployment

Reply:
![image](https://user-images.githubusercontent.com/73839461/143219823-4b8f7105-8d73-42d2-9f4a-7e8b4ec159a6.png)
This seems wrong though? Why suddenly 2 spaces after all the tabs?

Reply:
Not sure why - lint tool seems to be enforcing it.